lib/gems/pending/util/win32/miq-powershell.rb

```diff
@@ -94,7 +93,8 @@ def self.is_error_object?(xml)
   end
 
   def self.run_script(script)
-    encoded_command = MIQEncode.encode(script.AsciiToUtf8.Utf8ToUnicode, false)
+    string = script.dup.force_encoding("ISO-8859-1").force_encoding("UTF-8").encode("UTF-16LE")
```

Comment:
The intermediate encoding seems pointless.  Let's do

```suggestion
    string = script.dup.force_encoding("ISO-8859-1").encode("UTF-16LE")
```

Reply:
I was sort of baffled by it, but didn't want to risk breaking something. Ok, will update.

Reply:
It was probably to avoid a new `AsciiToUnicode` method since the other 2 were already available.